### PR TITLE
ci: build wheels

### LINKED ISSUE #1: wheels
as it's not clear for casual programmer but enthousiast end-user to make a wheel from source, would it be possible that the project to provide wheels directly ? at least for Windows ?

the more easy it is to play with it right now, the more it will be a flury of initiatives around it for 3.12.0 final

### PR DESCRIPTION
**Pull request by henryiii**
Needs #3. Closes #1. Assumes this repo gets set up with PyPI trusted publisher deployment.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -11,7 +11,7 @@ authors = [
 ]
 description = "Use this module to try out multiple interpreters and a per-interpreter GIL in Python 3.12.  Do not use this for anything important."
 readme = "README.md"
-requires-python = "=3.12"
+requires-python = ">=3.12,<3.13"
```
Comment (henryiii) [inline pyproject.toml:14]: I'm not a fan of `<`, especially for `requires-python`, but this is a valid Specifier, while `=` is not.

File: pyproject.toml [hunk 2]
```diff
@@ -71,3 +71,14 @@ ignore = [
   "S101",  # "assert" is OK
   "E702",  # Statements on one line OK
 ]
+
+[tool.cibuildwheel]
+skip = ["pp*"]
+# Trigger an install of the package, but run nothing of note
+test-command = "echo Wheel installed"
```
Comment (henryiii) [inline pyproject.toml:80]: The actual tests lock up (also seen in #5). Issue at #6.

**Comment by henryiii** [comment]
Successful run:

https://github.com/henryiii/interpreters/actions/runs/6409787093/job/17401721393

I've skipped 32-bit Windows, and the tests don't run, it just verifies that it is importable (hangup issue seen in #5 & pointed out in #6.)

Also I realize I didn't include Windows ARM. I can do that, but ideally I should also test it locally, which takes a bit of time to setup.

I guess I could also add Linux special archs if you'd like.

**Comment by stonebig** [comment]
it seems we can build easily a wheel now from latest interpreters_pep_734 source, and python-3.13.0b1

**Comment by henryiii** [comment]
Yes, it’s not a binary any more. Spoke with Eric here at PyCon.

**Pull request closed without merging**